.github/pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+## Changes
+
+_Explanation of changes goes here_
+
+## Notes
+
+_Any other notes go here_
+
+## To Do
+
+_Any remaining things that need to get done_
+
+- [ ] item 1
+- [ ] ...
+
+## Checklist
+
+It can be helpful to check the `Checks` and `Files changed` tabs.
+Please review the [contributor guide](https://github.com/Northeastern-Electric-Racing/PM-Dashboard-v2/blob/main/docs/ContributorGuide.md) and reach out to your squad if anything is unclear.
+
+- [ ] All commits are tagged with the ticket number
+- [ ] No linting errors
+- [ ] No newline at end of file warnings
+- [ ] All code follows repository-configured prettier formatting
+- [ ] No merge conflicts
+- [ ] All checks passing
```

Comment:
This feels a bit repetitive -- if you have a checkbox for the tests, linting errors, and merge conflicts, then this is implicitly true. Is there a reason why this checkbox was added?

Reply:
I agree it might be repetitive, also with GitHub preventing merges if checks fail, but I do think we should consider taking into account possible future checks we might add, so maybe rephrasing it or putting it somewhere else?

Reply:
one part of this checklist is making it abundantly obvious to new devs how to interpret the GitHub UI. People ask for reviews on PRs that don't have all checks passing, so this is me saying "fix it first pls". My intention here is that I rarely want to review a PR before everything in the checklist is checked, but tech leads certainly should be assisting devs in getting their PRs ready